src/styles/index.scss

```diff
@@ -1,3 +1,73 @@
+$days: 31;
+$month-length: 28, 29, 30, 31;
+$start-days: (
+  'mon': 0,
+  'tue': 101px,
+  'wed': 202px,
+  'thu': 303px,
+  'fri': 404px,
+  'sat': 505px,
+  'sun': 606px,
```

Comment:
The values for the `$start-days` map use hard-coded pixel values. Consider using a base unit or a calculation based on a single unit to ensure consistency and maintainability. For example, if the base unit is 101px, you can use multiples of that unit for each day.